todo-app/src/components/TodoItem.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="flex flex-row sm:flex-col justify-between items-center">
+    <div class="flex flex-row sm:justify-between sm:w-full">
+      <div class="font-semibold text-lg sm:text-4xl">
+        {{ props.todo.title }}
+      </div>
+      <div
+        class="w-2 h-2 ml-2 mt-3 sm:w-32 sm:h-10 rounded-full flex items-center justify-center"
+        :class="colorMap[props.todo.importance]"
+      >
+        <div class="hidden sm:block text-lg font-semibold text-white">
+          {{ props.todo.importance }}
+        </div>
+      </div>
+    </div>
+    <div
+      class="flex flex-col sm:flex-row justify-between sm:pt-6 sm:mt-6 sm:w-full"
+    >
+      <div class="text-neutral-500 hidden sm:block sm:text-2xl font-semibold">
+        {{ props.todo.content }}
+      </div>
+      <button class="relative" @click="handleCheckClick">
+        <GreenEllipseIcon class="w-6 sm:w-10" v-if="checked" />
```

Comment:
wrap GreenEllipseIcon and GreenCheckIcon inside a template tag and put the v-if to that, also the v-if should always be in the first position